tests/unit/services/secrets-manager.unitTest.ts

```diff
@@ -2,8 +2,9 @@ import { GetSecretValueCommand, SecretsManagerClient } from '@aws-sdk/client-sec
 import { mockClient } from 'aws-sdk-client-mock';
 import { getSecretValue } from '../../../src/services/secrets-manager';
 
-const mockSecretsManager = mockClient(SecretsManagerClient);
+const mockSecretsManager = mockClient(SecretsManagerClient as any);
```

Comment:
Why is the cast as any needed? Same for the ts ignore?

Reply:
These were required after resolving the dependency vulnerabilities and bumping the later version of the sdk.

Reply:
I've removed them, and i dont seem to be having any issues

Reply:
@LGin-BJSS @m-mullen 

Any chance you can try this too please? I've just taken these out, deleted the build directory and rebuilt just in case - but throwing an error.

Reply:
I also get a compilation error when this is reverted